src/main/java/core/basesyntax/service/ReportCheck.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.Map;
+
+public interface ReportCheck {
+    void readFromReport(String filName, Map<Procedure, OperationStrategy> operationStrategyMap);
```

Comment:
```suggestion
    void readFromReport(String fileName, Map<Procedure, OperationStrategy> operationStrategyMap);
```
Let's add a map via constructor

Reply:
done